docs/assets/css/homepage.scss

```diff
@@ -3,7 +3,7 @@
 
 // Imports -------------- //
 
-@import "us_web_design_standards";
+@import 'uswds';
```

Comment:
It's coming from this part of the `copy-assets` task here => https://github.com/18F/web-design-standards/pull/1035/files#diff-150b96ca2a378526f6a28dfc8c3e7687R44